src/service/video-manager-impl.js

```diff
@@ -405,6 +478,30 @@ class VideoEntry {
       mutate,
     });
   }
+
+
+  /**
+   * Returns whether the video is paused or playing after the user interacted
+   * with it or playing through autoplay
+   * @return {!../video-interface.VideoInterface} PlayingStates
+   */
+  getPlayingState() {
+    if (this.userInteracted_ && this.isPlaying_) {
```

Comment:
Wouldn't it be better to put it `this.playCalledByAutoplay_ ` inside `play()`?
Also if the user didn't interact and the video is playing, doesn't that inherently mean that autoplay is supported? not sure what else could cause such behavior.